### DISCUSSION TITLE
Expo-like Continuous Native Generation

### DISCUSSION THREAD

**Discussion by RichiCoder1** [ionic-team/capacitor#7083] [Ideas]
I've been writing more Capacitor apps recently, and one thing I miss from Expo/RN is [Expo's Config plugins](https://docs.expo.dev/config-plugins/introduction/). These are incredibly nice and powerful because they allow you to entirely omit native assets/folders from your repo and focus entirely on the application itself. While also providing an easier to maintain escape hatch to do native modifications.

Is there any sort of equivalent for Capacitor?

**Comment by RichiCoder1** [comment]
Whelp, shortly after posting this I found https://trapeze.dev/ which seems a close-ish approximation. Is that project still being actively developed?

**Comment by rommyarb** [comment]
Cool idea! I think the other framework like NativeScript has `nativescript.config.ts` to do this
